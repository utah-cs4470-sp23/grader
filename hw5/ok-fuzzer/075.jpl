show 63
assert - false, "="
print "'&"
read image "@" to I_.5_
read image "*" to x66
let K5 = false
read image "+" to F
show true >= []
show 4
read image ",_%" to q0l
time assert - O ( ) [ - false , Ir , [] , array [ ] GO. , true ], ""
read image "/" to Xr_.
read image "" to b
print ""
read image "" to B_t6_
