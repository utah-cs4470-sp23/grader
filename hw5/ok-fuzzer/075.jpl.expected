(ShowCmd (IntExpr 63))
(AssertCmd (UnopExpr - (FalseExpr)) "=")
(PrintCmd "'&")
(ReadCmd "@" (VarArgument I_.5_))
(ReadCmd "*" (VarArgument x66))
(LetCmd (ArgLValue (VarArgument K5)) (FalseExpr))
(ReadCmd "+" (VarArgument F))
(ShowCmd (BinopExpr (TrueExpr) >= (ArrayLiteralExpr)))
(ShowCmd (IntExpr 4))
(ReadCmd ",_%" (VarArgument q0l))
(TimeCmd
 (AssertCmd
  (UnopExpr
   -
   (ArrayIndexExpr
    (CallExpr O)
    (UnopExpr - (FalseExpr))
    (VarExpr Ir)
    (ArrayLiteralExpr)
    (ArrayLoopExpr (VarExpr GO.))
    (TrueExpr)))
  ""))
(ReadCmd "/" (VarArgument Xr_.))
(ReadCmd "" (VarArgument b))
(PrintCmd "")
(ReadCmd "" (VarArgument B_t6_))
